Prüfschritte/de/11.1.4.4 Text auf 200% vergrößerbar.adoc

```diff
@@ -42,17 +42,10 @@ Muss nur geprüft werden, wenn die Systemfunktion zum vergrößern nicht vorhand
 
 === 3. Hinweise
 
-Werden Apps getestet, kommen die Funktionen Zoomen und / oder Nur-Text-Vergrößerung des Betriebssystems zum Einsatz:
+* Die Zoomfunktion (Vergrößerung des gesamten Inhalt des Bildschirms) des Betriebssystems wird in der Regel eine 200%ige Vergrößerung ermöglichen. Funktioniert die Zoomfunktion mit der zu testenden Software, ist dieser Prüfschritt bestanden. 
+Native mobile Plattformen verfügen über eine Funktion "größerer dynamischer Text" das bedeutet, eine Funktion zur Textvergrößerung. Apps, die diese Funktion unterstützen, passen sich an die festgelegte Lesegröße an. Diese Funktion wird häufig verwendet, da die Bedürfnisse von sehbehinderten Nutzenden durch die Zoom-Funktion der Plattform nicht vollständig abgedeckt werden. Wir empfehlen daher dringend, native Apps so umzusetzen, dass sie diese Funktion unterstützen. Probleme die bei der Nutzung der dynamischen Textfunktion entstehen, sollten zumindest angemerkt werden.
```

Comment:
Komma nach "größerer dynamischer Text"
Apps, die dynamische Fonts verwenden, passen diese an die eingestellte Schriftgröße an.

Bei dem Rest bin ich mit noch unsicher, ob wir das so wirklich drinlassen sollten. Warumgerade hier dringende Empfehlung an App-Entwickler, in einem Prüfschritt? Sollen wir dann auch schreiben: Wir empfehlen dringend, Überschriften einzusetzen, die Tastaturreihenfolge richtig umzusetzen usw. usf.? 

Und was genau kann man sich unter "Probleme, die bei der Nutzung der dynamischen Textfunktion entstehen" vorstellen? Dynamische Fonts in Toolbars, so dass Texte abgeschnitten werden oder Dinge komisch umbrechen? Oder umgekehrt, manche Inhalte vergößern dynamisch, andere nicht?

Wenn's dinbleibt: Probleme (Komma) die...